src/renderer/components/+preferences/add-helm-repo-dialog.tsx

```diff
@@ -27,22 +27,31 @@ enum FileType {
 
 @observer
 export class AddHelmRepoDialog extends React.Component<Props> {
-  private emptyRepo = {name: "", url: "", username: "", password: "", insecureSkipTlsVerify: false, caFile:"", keyFile: "", certFile: ""};
+  static metadata = observable({
+    isOpen: false,
```

Comment:
Just wondering why we have to do these like this?

Reply:
What exactly? Keeping observable state for single dialog instance? 
Not sure why about current dialog, but for example for `ConfirmDialog` observable state is required (it's rendered once in `[lens-]app.tsx`)

Reply:
I meant that previously we had `@observable isOpen = false`, now we wrap these in `metadata`.

Reply:
Because of `static` field. It's not possible to use it with `@observable` decorator.

Other option: `static isOpen = observable.box(false)`, but for consistency with other dialogs it's just an observable object with some metadata.